src/fsharp/FSharp.Core.Unittests/SurfaceArea.coreclr.fs

```diff
@@ -293,6 +293,222 @@ Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityC
 Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityComparer`1[T] Structural[T]()
 Microsoft.FSharp.Collections.HashIdentity: System.String ToString()
 Microsoft.FSharp.Collections.HashIdentity: System.Type GetType()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Boolean Equals(System.Object)
```

Comment:
Big question: how much of this stuff needs to be public to get the performance gains?  Is it already minimized?